core/services/chainlink/config_test.go

```diff
@@ -379,6 +379,11 @@ func TestConfig_Marshal(t *testing.T) {
 					LimitDefault:       ptr[uint32](12),
 					LimitMultiplier:    mustDecimal("1.234"),
 					LimitTransfer:      ptr[uint32](100),
+					OCRJobLimit:        ptr[uint32](10),
+					DRJobLimit:         ptr[uint32](10),
+					VRFJobLimit:        ptr[uint32](10),
+					FMJobLimit:         ptr[uint32](10),
+					KeeperJobLimit:     ptr[uint32](10),
```

Comment:
Minor point, but could be worth using a different value for each one to ensure each is being set correctly

Reply:
Fixed